.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.log
 lib
 es
+.idea
```

Comment:
@DevSide I'm agreed with @D34THWINGS, IDE files shouldn't be part of the project's `.gitignore` 😞